Mono.Debugging/Mono.Debugging.Client/Catchpoint.cs

```diff
@@ -45,7 +46,14 @@ public Catchpoint (string exceptionName, bool includeSubclasses)
 			this.exceptionName = exceptionName;
 			this.includeSubclasses = includeSubclasses;
 		}
-		
+
+		public Catchpoint (string exceptionName, bool includeSubclasses, bool everythingElse)
+		{
```

Comment:
Should the other .ctors chain to this one rather than duplicating the initialization logic?